sandbox.html

```diff
@@ -180,11 +184,20 @@
   </div>
   <div>
     <label for="geometry">Geometry (GeoJSON)</label>
-    <textarea name="geometry" id="geometry" rows="5"></textarea>
+    <textarea name="geometry" id="geometry" rows="5">
+{
+  "type": "Point",
+  "coordinates": [8.682127, 50.1109238]
+}
+</textarea>
```

Comment:
Please do not add coordinate sample data again. I removed them, because they depend on the repo/map you use and do not make sense if that uses another srs (most of the time).

Instead, made the sandbox persist previous inputs per command, so that you don't need to enter them again and again.